action.yaml

```
@@ -95,6 +98,9 @@ runs:
         description=$(jq --raw-output '.description' "${{ steps.find.outputs.config }}")
         echo "::set-output name=description::${description}"
 
+        image=$(jq --raw-output '.image' "${{ steps.find.outputs.config }}")
```

`image` doesn't have to be there, to prevent is becoming `null` instead of an empty string:

```suggestion
        image=$(jq --raw-output '.image // empty' "${{ steps.find.outputs.config }}")
```